examples/basic/annotations/whisker.py

```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between Car class and HWY from the autommpg sample data. 
+This example demonstrates the use of Whisker to display quintile ranges in the plot.
```

Comment:
```suggestion
This example demonstrates the use of Whisker to display quantile ranges in the plot.
```

This typo also exists in the title of the figure and should be corrected there as well.

Reply:
Thank you for this correction @mosc9575